src/MainWindow/main_window.cpp

```diff
@@ -177,7 +177,15 @@ void MainWindow::openFileSlot() {
 }
 
 void MainWindow::newDesignCreated(const QString& design) {
-  setWindowTitle(m_projectInfo.name + " - " + design);
+  QFileInfo path(design);
+  QString strName = path.baseName();
+  QString fileName = path.fileName();
+  qDebug() << "strNme = " << strName;
+  if (strName == "") {
+    setWindowTitle(m_projectInfo.name + "-");
+  } else {
+    setWindowTitle(fileName + " - " + strName + " - " + m_projectInfo.name);
```

Comment:
const QFileInfo path(design);
setWindowTitle(path.fileName() + " - " + path.baseName() + " - " + m_projectInfo.name);

Reply:
Yeah I have fixed like this now.